docs/specs/xref.yml

```diff
@@ -43,7 +43,7 @@ inputs:
     { "references":[{"uid": "a", "name": "a", "fullName": "a", "href": "https://docs.microsoft.com/a", "nameWithType": "a"}]}
 outputs:
   docs/a.json: |
-    { "conceptual": "<p>Link to <a href=\"/en-us/dotnet/api/system.string\">System.String</a></p>\n"}
+    { "conceptual": "<p>Link to <a href=\"/dotnet/api/system.string\">System.String</a></p>\n"}
 ---
```

Comment:
I am not sure whether I've understood well on this PR, but should we keep the output JSON as unchanged because e2e behavior will still be appending locale.

Reply:
The e2e behavior for Docs will be the same, which will be covered by the regression tests, and the output file for Docs should be *.raw.page.json.

For this spec test, this output type is *.json, not pageJson, which have not applied template yet, and we will add locales during the applying template stage.

Reply:
Got it! Thanks for explanation!